core/scripts/cli/voting/batchCommitVotes.js

```diff
@@ -0,0 +1,38 @@
+const style = require("../textStyle");
```

Comment:
Have you thought about combining batchCommitVotes.js, batchRevealVotes.js, and batchRetrieveRewards.js into one file?

This has a few advantages:
- Combining similar code into the same file (they all fundamentally follow the same pattern).
- Naming the functions inside the file, so it's easier to tell what the function does without having to look back at the filename.
- Reducing the overall number of files a reader needs to look through to understand the flow of data in the voting module.

There are also good reasons to keep them separate, so if you think they should stay separate, that's cool too.

Reply:
So  I don't think that these methods can be combined into one because they are different enough

But I do agree they can be and should be combined into one file--in fact it can also be in the `common` file for use in other scripts

Reply:
Sorry if I miscommunicated -- yes, just the same file, I don't think it makes sense to make them all the same function or anything :)